x-pack/solutions/security/packages/features/src/assistant/product_feature_config.ts

```diff
@@ -33,4 +33,15 @@ export const assistantDefaultProductFeaturesConfig: Record<
       AssistantSubFeatureId.manageGlobalKnowledgeBase,
     ],
   },
+  [ProductFeatureAssistantKey.configurations]: {
```

Comment:
@tomsonpl Could you please confirm if this is correct? In order to use `!productFeatureKeys?.has(ProductFeatureAssistantKey.configurations)` in `plugin.tsx`, I had to add `configurations` to the `ProductFeatureAssistantKey` enum which then required a definition here.

Reply:
Sorry I put in incorrectly in the Slack, [this](https://github.com/elastic/kibana/pull/217373/files#diff-fa30014b439035af4cc4c875257fb2707ebbeb2f77acb875273129f163db013dR216) should be `ProductFeatureSecurityKey.configurations` Tomasz added in https://github.com/elastic/kibana/pull/216079/files#diff-feb597dea55daaccd3694cdd0820406bcbe729dc2477301c20a7e505668f4fc2R57

I think we can revert the change here.

Reply:
ok, reverted!